frontend/cypress/integration/todo.spec.ts

```diff
@@ -13,18 +13,44 @@ describe('todo', () =>
 
 	it('adding todo to list', () =>
 	{
-		cy.get('[data-test="item"]').last().should('not.contain.text', 'My new Todo from Cypress');
-		cy.get('[data-test="text-input"]').type('My new Todo from Cypress');
+		cy.get('[data-test="item"]').should('not.contain.text', 'My new Todo from Cypress');
+		cy.get('[data-test="title-input"]').type('My new Todo from Cypress');
+		cy.get('[data-test="description-input"]').type('Cypress Todo Described');
+		cy.get('[data-test="url-input"]').type('https://images.unsplash.com/photo-1647163927209-b7e5a9d53edc?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1632&q=80');
```

Comment:
Use random pictures - urls is shorter aswell: https://awik.io/generate-random-images-unsplash-without-using-api/